presto-cli/src/main/java/io/prestosql/cli/FormatUtils.java

```diff
@@ -114,7 +114,7 @@ else if (longForm) {
 
     public static String formatDataRate(DataSize dataSize, Duration duration, boolean longForm)
     {
-        double rate = dataSize.toBytes() / duration.getValue(SECONDS);
+        long rate = Math.round(dataSize.toBytes() / duration.getValue(SECONDS));
```

Comment:
This needs to stay as a double so that the NaN / infinity check below works (or restructure the code to account for the cases where divisor and possibly dividend is zero).